### PR TITLE
new class created Hel

### PR DESCRIPTION
**Pull request by taron19**

### CODE REVIEW

**Comment by taron19** [comment]
i added hel class and main method

**Comment by taron19** [comment]
hey do you see me

File: Hel.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class Hel {
+    public static void main(String[] args) {
+        System.out.println("new class");
+    }
```
Comment (taron1991) [inline Hel.java:4]: add new methos with array

Comment (taron1991) [reply Hel.java:4]: got it?

**Review by taron1991 — CHANGES_REQUESTED** [review]

File: Hel.java (hunk 1 above)
Comment (taron19) [reply Hel.java:4]: ok now

**Review by taron19 — COMMENTED** [review]
good

**Pull request merged**